### PR TITLE
[EPO-2795] GalaxiesSelector and GalacticProperties components support randomized data sources

### PR DESCRIPTION
**Pull request by blnkt**
Includes support for user generated data alongside random data source or sources;
Includes updated pageData implementing random/multi data sources;
Random data source saved as "hidden" answer (answer wo question);
Includes updated (compressed) hsc images;

### CODE REVIEW

File: src/containers/GalaxiesSelectorContainer.jsx [hunk 1]
```diff
@@ -33,7 +57,7 @@ class GalaxiesSelectorContainer extends React.PureComponent {
       this.setState(prevState => ({
         ...prevState,
         data: objects,
-        imagePath: `/images/galaxies/hsc/${name}.png`,
+        imagePath: `/images/galaxies/hsc/${name}.jpg`,
```
Comment (georgegomez144) [inline src/containers/GalaxiesSelectorContainer.jsx]: I'm understanding that from this, there will always be a compressed jpg file format and not a png?

File: src/state/GlobalStore.js [hunk 2]
```diff
@@ -124,7 +124,18 @@ class GlobalStore {
...
-        const { questions, answers: prevAnswers } = prevIPS[pageId];
+        const prevPageIPS = prevIPS[pageId];
+
+        if (!prevIPS[pageId]) {
+          return global;
+        }
+
+        const { questions, answers: prevAnswers } = prevPageIPS;
+
+        if (!questions[qId]) {
```
Comment (georgegomez144) [inline src/state/GlobalStore.js]: On the page `/observable-universe/exploring-the-deep/`, I'm getting an error in the console (page not loading futher) that says, "Cannot read property 'randomGalaxies' of null" that points to this line.

File: src/containers/GalaxiesSelectorContainer.jsx (hunk 1 above)
Comment (blnkt) [reply src/containers/GalaxiesSelectorContainer.jsx]: sure.  In this case the pngs were just way too big and were slowing page loads down so I elected to update the files used.

File: src/state/GlobalStore.js (hunk 2 above)
Comment (blnkt) [reply src/state/GlobalStore.js]: I'm not able to reproduce the issues you're describing.  Are you on the latest commit? Can you provide instructions to reproduce `"Cannot read property 'randomGalaxies' of null"` and `/observable-universe/exploring-the-deep-1/ is 'stuck' loading the page. Only the spinner is loading on the selector.`?

File: src/containers/GalaxiesSelectorContainer.jsx (hunk 1 above)
Comment (georgegomez144) [reply src/containers/GalaxiesSelectorContainer.jsx]: 👏 Good decision, then!!!!

File: src/state/GlobalStore.js (hunk 2 above)
Comment (georgegomez144) [reply src/state/GlobalStore.js]: No specific instructions other than go to the page. I did nothing other than clear my local storage.

**Pull request merged**